src/nodes/ImageView.vue

```diff
@@ -280,18 +251,72 @@ export default {
 			this.loaded = true
 			return
 		}
-		const img = new Image()
-		img.onload = () => {
-			this.imageLoaded = true
-		}
-		img.onerror = () => {
+		this.init()
+	},
+	methods: {
+		init() {
+			if (this.src.startsWith('text://')) {
+				const imageFileName = getQueryVariable(this.src, 'imageFileName')
+				this.loadImage(this.getTextApiUrl(imageFileName)).catch((e) => {
+					this.onImageLoadFailure()
+				})
```

Comment:
You can simplify this code.

```suggestion
				this.loadImage(this.getTextApiUrl(imageFileName))
				  .catch(this.onImageLoadFailure)
```

Reply:
Also, we can use async/await and wrap `this.init()` and handle error from `this.init()`

```suggestion
				await this.loadImage(this.getTextApiUrl(imageFileName))
```

or

```suggestion
				return this.loadImage(this.getTextApiUrl(imageFileName))
```

and, change `this.init()` call.

```js
this.init().catch(this.onImageLoadFailure)
```